ui/components/src/Drawer/Drawer.tsx

```diff
@@ -20,7 +20,7 @@ export interface DrawerProps extends MuiDrawerProps {
   anchor?: 'left' | 'right';
 }
 
-const DRAWER_DEFAULT_WIDTH = 900;
+const DRAWER_DEFAULT_WIDTH = 1080;
```

Comment:
Making this a little wider to match updated designs and leave a little more breathing room for values in the chart view in the panel editor.